templates/contribute/naming.md

```diff
@@ -259,6 +259,40 @@ check le_of_mul_le_mul_left
 check le_of_mul_le_mul_right
 ```
 
+## Naming of structural lemmas ##
+
+We are trying to standardize certain naming patterns for structural lemmas.
+At present these are not uniform across mathlib.
+
+### Extensionality ###
+
+A lemma of the form `(∀ a, a ∈ s₁ ↔ a ∈ s₂) → s₁ = s₂` should be named `.ext`,
```

Comment:
We also have `∀ x, f x = g x` as a common assumption for `@[ext]`.

Reply:
```suggestion
A lemma of the form `(∀ x, f x = g x) → f = g` should be named `.ext`,
```